src/jvm/main/org/jetbrains/kotlinx/lincheck/strategy/Strategy.kt

```diff
@@ -90,6 +90,11 @@ abstract class Strategy protected constructor(
     override fun close() {
         runner.close()
     }
+
+    /**
+     * Restores recorded values of all memory reachable from static state.
+     */
+    open fun restoreStaticMemorySnapshot() {}
```

Comment:
Let's keep it only on the level of `ManagedStrategy` and do not expose on `Stategy` API level.